src/app/views/provisioning/order.html

```diff
@@ -12,12 +12,12 @@ <h1 translate>{{ vm.subscriptionPlanText }}</h1>
       </div>
 
       <div ng-hide="vm.isLoading" class="table-cell" ng-show="!isLoading">
-        <div class="row th-row hidden-xs">
-          <div class="col-xs-10 col-xs-offset-2">
-            <div class="cell col-sm-2">{{ 'mno_enterprise.templates.dashboard.provisioning.order.name' | translate }}</div>
-            <div class="cell col-sm-4">{{ 'mno_enterprise.templates.dashboard.provisioning.order.description' | translate }}</div>
-            <div class="cell col-sm-2">{{ 'mno_enterprise.templates.dashboard.provisioning.order.price' | translate }}</div>
-            <div class="cell col-sm-3">
+        <div class="row th-row">
+          <div class="col-xs-11 col-xs-offset-1">
+            <div class="cell col-sm-2 hidden-xs" >{{ 'mno_enterprise.templates.dashboard.provisioning.order.name' | translate }}</div>
+            <div class="cell col-sm-4 col-xs-offset-1 hidden-xs">{{ 'mno_enterprise.templates.dashboard.provisioning.order.description' | translate }}</div>
+            <div class="cell col-sm-3 hidden-xs">{{ 'mno_enterprise.templates.dashboard.provisioning.order.price' | translate }}</div>
+            <div class="cell col-xs-6 col-sm-2">
```

Comment:
Please keep the order consistent betweeen `sm` and `xs`.
Previous columns have `sm` before `xs`